test/datatype.test.js

```diff
@@ -220,12 +220,12 @@ describe('datatypes', function() {
 
     it('should convert property value undefined to null', function(done) {
       var EXPECTED = {desc: null, extra: null};
+      var data = {desc: undefined, extra: undefined};
       if (isStrict) {
         // SQL-based connectors don't support dynamic properties
         delete EXPECTED.extra;
+        delete data.extra;
```

Comment:
![https://c1.staticflickr.com/3/2777/4281285527_37862b7d7e_b.jpg](slap in the forehead)

Ah, of course! Makes a lot of sense.